### PR TITLE
fix: v8 Dropdown multiselect option hover contrast in WHCM

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

A non-HCM background style was overriding the background color in HCM, causing the text to be unreadable:
screenshot of an open dropdown, showing black text on a black background

## New Behavior

This PR resets the HCM colors for the multiselect hover selector, so it matches single-select hover (HighlightText on Highlight background)
screenshot of an open multiselect dropdown, showing visible text and a visible checkbox border on a highlighted item

## Related Issue(s)

- Fixes [15910](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/15910)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Avatar-mount_base.html">933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Avatar-mount.html">905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Button-mount_base.html">572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Button-mount.html">575</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Field-mount_base.html">1625</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Field-mount.html">1641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProvider-mount_base.html">1168</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProvider-mount.html">1141</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProviderWithTheme-mount_base.html">300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProviderWithTheme-mount.html">309</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">709</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">243</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">293</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/InfoButton-mount_base.html">209</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/InfoButton-mount.html">208</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/MakeStyles-mount_base.html">1342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/MakeStyles-mount.html">1333</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Persona-mount_base.html">6828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/Persona-mount.html">2257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/SpinButton-mount_base.html">1999</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test-react-components/SpinButton-mount.html">1967</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `67.495 kB`<br />`18.211 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `206.901 kB`<br />`57.339 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |    `38.32 kB`<br />`12.15 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.446 kB`<br />`2.185 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/be0ca69899300abe3c8478c435e0f6837138479b'>be0ca69899300abe3c8478c435e0f6837138479b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/BaseButton-mount_base.html">982</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/BaseButton-mount.html">990</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Breadcrumb-mount_base.html">2121</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Breadcrumb-mount.html">2354</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Checkbox-mount_base.html">2147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Checkbox-mount.html">2212</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/CheckboxBase-mount_base.html">1984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/CheckboxBase-mount.html">1819</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ChoiceGroup-mount_base.html">3810</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ChoiceGroup-mount.html">3861</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ComboBox-mount_base.html">1045</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ComboBox-mount.html">1010</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/CommandBar-mount_base.html">8240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/CommandBar-mount.html">8148</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ContextualMenu-mount_base.html">16472</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ContextualMenu-mount.html">16252</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DefaultButton-mount_base.html">1119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DefaultButton-mount.html">1164</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DetailsRow-mount_base.html">3017</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DetailsRow-mount.html">3000</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DetailsRowFast-mount_base.html">2809</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DetailsRowFast-mount.html">3053</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DetailsRowNoStyles-mount_base.html">2554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DetailsRowNoStyles-mount.html">2820</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Dialog-mount_base.html">3323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Dialog-mount.html">3240</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DocumentCardTitle-mount_base.html">435</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/DocumentCardTitle-mount.html">448</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Dropdown-mount_base.html">2687</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Dropdown-mount.html">2522</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/FocusTrapZone-mount_base.html">1559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/FocusTrapZone-mount.html">1497</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/FocusZone-mount_base.html">1581</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/FocusZone-mount.html">1607</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedList-mount_base.html">49195</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedList-mount.html">56920</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedList-virtual-rerender_base.html">23914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedList-virtual-rerender.html">24183</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">74356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedList-virtual-rerender-with-unmount.html">74718</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedListV2-mount_base.html">445</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedListV2-mount.html">406</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedListV2-virtual-rerender_base.html">389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedListV2-virtual-rerender.html">413</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">443</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">451</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/IconButton-mount_base.html">1593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/IconButton-mount.html">1424</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Label-mount_base.html">561</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Label-mount.html">576</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Layer-mount_base.html">3655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Layer-mount.html">3382</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Link-mount_base.html">669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Link-mount.html">594</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/MenuButton-mount_base.html">1393</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/MenuButton-mount.html">1385</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/MessageBar-mount_base.html">28794</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/MessageBar-mount.html">27921</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Nav-mount_base.html">2481</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Nav-mount.html">2687</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/OverflowSet-mount_base.html">1023</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/OverflowSet-mount.html">1133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Panel-mount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Panel-mount.html">2127</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Persona-mount_base.html">992</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Persona-mount.html">1049</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Pivot-mount_base.html">1350</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Pivot-mount.html">1352</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/PrimaryButton-mount_base.html">1243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/PrimaryButton-mount.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Rating-mount_base.html">5479</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Rating-mount.html">5840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SearchBox-mount_base.html">1184</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SearchBox-mount.html">1259</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Shimmer-mount_base.html">2269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Shimmer-mount.html">2427</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Slider-mount_base.html">1818</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Slider-mount.html">1614</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SpinButton-mount_base.html">3878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SpinButton-mount.html">3878</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Spinner-mount_base.html">646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Spinner-mount.html">567</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SplitButton-mount_base.html">2663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SplitButton-mount.html">2566</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Stack-mount_base.html">599</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Stack-mount.html">633</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/StackWithIntrinsicChildren-mount_base.html">1174</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/StackWithIntrinsicChildren-mount.html">1211</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/StackWithTextChildren-mount_base.html">3306</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/StackWithTextChildren-mount.html">3313</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SwatchColorPicker-mount_base.html">8058</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/SwatchColorPicker-mount.html">8798</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/TagPicker-mount_base.html">2042</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/TagPicker-mount.html">1991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Text-mount_base.html">638</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Text-mount.html">630</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/TextField-mount_base.html">1249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/TextField-mount.html">1331</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ThemeProvider-mount_base.html">1239</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ThemeProvider-mount.html">1226</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ThemeProvider-virtual-rerender_base.html">782</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ThemeProvider-virtual-rerender.html">904</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1723</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Toggle-mount_base.html">904</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/Toggle-mount.html">913</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/buttonNative-mount_base.html">408</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27306/perf-test/buttonNative-mount.html">382</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-4cf263e9-5a59-4ce0-b9b1-a08d8baa5e67.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-4cf263e9-5a59-4ce0-b9b1-a08d8baa5e67.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27306&commitId=0f3ccc52d0e4d4b7af7d550cfd6b90814df44d3b&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by TristanWatanabe — APPROVED** [review]

**Pull request merged**